.github/workflows/build.yml

```diff
@@ -20,11 +20,14 @@ jobs:
         uses: actions/setup-go@v4
         with:
           go-version: ^1.22
-
-      - name: Build
-        run: make build
+      - name: Install golangci-lint
+        uses: golangci/golangci-lint-action@v6
+        with:
+          version: v1.59
```

Comment:
Tip: You could have used "latest" so you don't have to maintain it

Reply:
fyi, Same reply from the other comment, but i'll throw in tools like renovate exist so you can automate updating things, while keeping consistent build experience.

Reply:
I'm sure dependabot cannot touch this argument of an action. It can bump the action version. But not its argument.

But maybe renovate can do it, so it would be fine